src/Surfnet/StepupGateway/GatewayBundle/Service/GlobalViewParameters.php

```diff
@@ -55,8 +55,12 @@ public function __construct(TranslatorInterface $translator, array $locales, arr
     /**
      * @return string
      */
-    public function getSupportUrl()
+    public function getSupportUrl(): string
     {
-        return $this->supportUrl[$this->translator->getLocale()];
+        $locale = $this->translator->getLocale();
+        if (array_key_exists($locale, $this->supportUrl)) {
+            return $this->supportUrl[$locale];
+        }
+        return '';
```

Comment:
The application no longer breaks on a missing language in the `stepup_language` cookie.